cmd/up/exec_unix.go

```diff
@@ -0,0 +1,41 @@
+//go:build !windows
+// +build !windows
+
+package up
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"syscall"
+)
+
+func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
+	var c *exec.Cmd
+	if runtime.GOOS != "windows" {
+		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
```

Comment:
I believe we're not passing the `ctx` to `exec.Command` to avoid sending `SIGKILL` and handle gracefully the shutdown of child processes?